apps/voting-disputable/app/src/components/VoteDetail/DisputablePeriod.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { GU, Timer, useTheme } from '@aragon/ui'
+import { format } from 'date-fns'
+
+//48 hour period
+const PERIOD = 1000 * 60 * 60 * 48
+const formatDate = date => `${format(date, 'yyyy-MM-dd, HH:mm')}`
```

Comment:
We're using this format in a few places now, perhaps it's worth moving it into `utils` and updating references to it?